### DISCUSSION TITLE
Provider for San Marino support

### DISCUSSION THREAD

**Discussion by lgulbinauskas** [azuyalabs/yasumi#352] [General]
Hi, @stelgenhof At the moment there's no provider for San Marino, ISO3166_2 - SE.
In the mean time I've mapped to use Italy since San Marino is landlocked inside it. Obviously it has seperate holidays from Italy, thus additional provider is needed.

**Comment by stelgenhof** [comment]
Hi @lgulbinauskas That's great! Do you think you can make a PR for San Marino?

**Comment by stelgenhof** [comment]
@lgulbinauskas It has been a while, and thought you may be interested to know I've created a provider for San Marino. You can check the feature branch of it here: https://github.com/azuyalabs/yasumi/tree/feature-add-san-marino
